lib/python/rose/gtk/__init__.py

```diff
@@ -0,0 +1,8 @@
+try:
+    import pygtk
+    pygtk.require('2.0')
+    import gtk
+except (ImportError, RuntimeError):
```

Comment:
``.require()`` seems to throw an ``AssertionError`` when the 'required version is not found on system', so shouldn't this be added to exceptions?

Reply:
Good point!